source/catalog.html

```diff
@@ -37,11 +40,11 @@
 
     <main class="page__main page-main">
       <div class="page-main__wrapper">
-        <h1 class="main-title">Каталог продукции</h1>
+        <h1 class="page-main__title main-title page-main__title--catalog">Каталог продукции</h1>
```

Comment:
Странный порядок классов. Раз уж сперва идет элемент, то и его модификатор должен быть следом
```suggestion
        <h1 class="page-main__title page-main__title--catalog main-title">Каталог продукции</h1>
```